server/cmwell-dc/src/main/scala/cmwell/dc/stream/SingleMachineInfotonIngester.scala

```diff
@@ -59,7 +59,13 @@ object SingleMachineInfotonIngester extends LazyLogging {
         case (infotonSeq, state) => {
           val payloadBuilder = new ByteStringBuilder
           // no need for end line because each line in already suffixed with it
-          infotonSeq.foreach(payloadBuilder ++= _.data)
+          infotonSeq.foreach{i=>
+            val firstLine = i.data.takeWhile(_ != stream.newLine)
+            if(i.data.takeWhile(_ != stream.newLine).contains(ByteString("meta/sys#uuid")))
```

Comment:
1. you create the same ByteString for each infoton - make a val for it.
2. i.data.takeWhile(_ != stream.newLine) is done twice - make a val
3. contains seems to not be working in this case

Reply:
1. I compare to string so it's not relevant in the new impl
2.done
3.done